code/game/objects/structures/light_pole.dm

```diff
@@ -13,6 +13,21 @@
 /obj/structure/light_pole/r
 	icon_state = "rustlamp_r"
 
+/obj/structure/light_pole/decayed
+	name = "light pole"
+	desc = "A tall light source. The bulb appears to be decayed."
+	icon = 'icons/effects/32x96.dmi'
+	icon_state = "rustlamp_l"
+	anchored = TRUE
+	density = TRUE
+	light_wedge = LIGHT_OMNI
+	light_color = LIGHT_COLOR_DECAYED
+	light_range = 8
+	light_power = 8
```

Comment:
do not re-define all these vars, they're already defined in the parent type
only re-define what is changed from the parent type, in this case desc and light_color

Reply:
I think this is right now.